research/src/pages/select/select.proposal.mdx

```diff
@@ -116,11 +116,55 @@ children:
+| `keydown(up key)`\*   | Moves focus to the previous `<option>` in the listbox                                                                  | focus                                                                    |
+| `keydown(enter)`      | Changes the `selected` state of the current `<option>` and updates the `<select>` value property                       | [selected](#selected) prop <br/> [value](#value) prop                    |
+| `keydown(space)`      | Changes the `selected` state of the current `<option>` and updates the `<select>` value property                       | [selected](#selected) prop <br/> [value](#value) prop                    |
+| `keydown(enter)`      | If the `<select>` does **not** have the `multiple` attribute then toggle the state of `open` on the `<select>`         | [open](#open-state) state                                                |
+| `keydown(enter)`      | If the `<select>` does **not** have the `multiple` attribute then toggle the aria-expanded attribute on the `<select>` | [aria-expanded](#https://www.w3.org/TR/wai-aria-1.1/#aria-expanded) attr |
+| `keydown(space)`      | If the `<select>` does **not** have the `multiple` attribute then toggle the state of `open` on the `<select>`         | [open](#open-state) state                                                |
+| `keydown(space)`      | If the `<select>` does **not** have the `multiple` attribute then toggle the aria-expanded attribute on the `<select>` | [aria-expanded](#https://www.w3.org/TR/wai-aria-1.1/#aria-expanded) attr |
+| `keydown(escape)`     | Toggles the `open` state of the `<select>`                                                                             | [open](#open-state) state                                                |
+
+### part option
+
+| Event       | Behavior                                                                                                       | Impacts                                               |
+| ----------- | -------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------- |
+| `click`     | Changes the `selected` state of the current `<option>` and updates the `<select>` value property               | [selected](#selected) prop <br/> [value](#value) prop |
+| `click`     | If the `<select>` does **not** have the `multiple` attribute then toggle the state of `open` on the `<select>` | [open](#open-state) state                             |
+| `pointerup` | Changes the `selected` state of the current `<option>` and updates the `<select>` value property               | [selected](#selected) prop <br/> [value](#value) prop |
+| `pointerup` | If the `<select>` does **not** have the `multiple` attribute then toggle the state of `open` on the `<select>` | [open](#open-state) state                             |
```

Comment:
We should probably have rows for `aria-expanded` here too, unless we just tie `open` to `aria-expanded` per my suggestion in another comment.